docs/sql-ref-syntax-ddl-create-function.md

```diff
@@ -74,7 +74,6 @@ CREATE [ OR REPLACE ] [ TEMPORARY ] FUNCTION [ IF NOT EXISTS ]
     * Should extend `UDF` or `UDAF` in `org.apache.hadoop.hive.ql.exec` package.
     * Should extend `AbstractGenericUDAFResolver`, `GenericUDF`, or
       `GenericUDTF` in `org.apache.hadoop.hive.ql.udf.generic` package.
-    * Should extend `UserDefinedAggregateFunction` in `org.apache.spark.sql.expressions` package.
```

Comment:
Shall we mention `Aggregator` as it's the new API to define native Spark UDAFs?

Reply:
Thanks for the review. `Aggregator` is explained in https://spark.apache.org/docs/3.1.2/sql-ref-functions-udf-aggregate.html and it is not directly relevant to `CREATE FUNCTION` because we cannot have the implementing class for `class_name` extends `Aggregator`, like `UserDefinedAggregateFunction`. So I don't think it should be mentioned here.

Reply:
can't we register `UserDefinedAggregator` via `CREATE FUNCTION`?

Reply:
Oh, wait.. seems like we can't.  Can we make it support `Aggregator`?

Reply:
Yea, we should provide a new option if we deprecate something. @beliefer do you have time to look into it?

Reply:
@cloud-fan Thank you for your ping.